temba/flows/models.py

```diff
@@ -5297,10 +5309,24 @@ def from_json(cls, org, json_obj):
             raise FlowException("Invalid reply action, no message")
 
         return cls(json_obj.get(cls.UUID), msg=json_obj.get(cls.MESSAGE), media=json_obj.get(cls.MEDIA, None),
-                   send_all=json_obj.get(cls.SEND_ALL, False))
+                   quick_replies=json_obj.get(cls.QUICK_REPLIES), send_all=json_obj.get(cls.SEND_ALL, False))
 
     def as_json(self):
-        return dict(type=self.TYPE, uuid=self.uuid, msg=self.msg, media=self.media, send_all=self.send_all)
+        return dict(type=self.TYPE, uuid=self.uuid, msg=self.msg, media=self.media, quick_replies=self.quick_replies,
+                    send_all=self.send_all)
+
+    @staticmethod
+    def get_translated_metadata(metadata, run):
+        """
+        Gets the appropriate metadata translation for the given contact
+        """
+        language_metadata = []
+        for item in metadata:
+            quick_reply = item.get(run.contact.language) \
```

Comment:
Should be using `run.flow.get_localized_text()` which is smart about how things backdown when languages are missing.